<html>
  <head>
    <style>
      body {background-color:white}
      h1   {color:#7D3364}
      h2   {color:black}
      h3   {color:#990066}
      .boxed {
      margin: 4px;
      border: 1px solid purple;
      background: lightgrey;
      color:black ;
      font:arial;
      }
    </style>

    <meta http-equiv="content-type" content="text/html; charset=utf-8">
<!--    <meta http-equiv="content-type" content="text/html; charset=windows-1252">//-->
  </head>

  <body>

    <h1><b>OrthoDB API</b></h1>

    The OrthoDB data can be programatically accessed using a URL based interface.
    In our implementation this means that the data can be retrieved using the following URL:
    <div class="boxed">
      <pre>
        <i>http://www.orthodb.org/CMD?ARG1="value"&ARG2="value&..."</i>
      </pre>
    </div>
    
    where <i>CMD</i> is a command and all <i>ARGx</i> are arguments to that specific command.
    Below follows a description of the available commands with arguments.

    <h2>Data Formats</h2>
    <p>
      All data is returned in <a href="https://en.wikipedia.org/wiki/JSON" >JSON</a> format, except for <b>/fasta</b> and <b>tab</b>.
      JSON data is widely supported by many languages. An overview with many examples can be found <a href="http://www.tutorialspoint.com/json"><b>here</b></a>.
    </p>
    <p>
      The JSON returned is of the generic format:
      <div class="boxed">
        <pre>
          {
             "url"    : full url of request
             "message": message string if status is error
             "status" : "ok" or "error"
             "data"   : array of data
          }
        </pre>
      </div>
    </p>

    <p>
      The clusters and genes have OrthoDB specific ids.
      <dl><b>Cluster id</b>
        <dt>Generic form <i>FFFVVCCCCII</i></dt>
        <dd><i>FFF</i> either EOG (eukaryota) or POG (prokaryota)</dd>
        <dd><i>VV</i> OrthoDB version (current '09')</dd>
        <dd><i>CCCC</i> unique identifier for each clade</dd>
        <dd><i>II</i> unique cluster identifier within the clade clade</dd>
      </dl>
      Example: <i>EOG091G06KN</i>

      <dl><b>Gene id</b>
        <dt>Generic form <i>taxid:geneid</i></dt>
        <dd><i>taxid</i>  is the NCBI taxonomy id</dd>
        <dd><i>geneid</i> is a unique zero-padded hexadecimal identifier</dd>
      </dl>
      Example: <i>10090:000d08</i>
      
    </p>

    <h2>Using the API</h2>
    <p>
      Interacting with the API can be done using either any web browser or a command line tool like 'wget' or 'curl'.
      <ul>
	<li>Linux: normally both are installed by default</li>
	<li>Windows: <a href=http://gnuwin32.sourceforge.net/packages/wget.htm>wget</a> and <a href=https://curl.haxx.se/download.html>curl</a></li>
	<li>Mac: 'curl' is usually installed natively, otherwise look <a href=https://curl.haxx.se/download.html>here</a></li>
      </ul>
    </p>
    <p>
      Example download fasta for a certain query and save in file 'data.fs' :
      <div class="boxed">
	<pre>
          <i>wget 'http://www.orthodb.org/fasta?query=Cytochrome%20P450&level=33208&species=33208' -O data.fs</i>
          <i>curl 'http://www.orthodb.org/fasta?query=Cytochrome%20P450&level=33208&species=33208' -o data.fs</i>
	</pre>
      </div>
      Note the difference in options for specifying output file.
    </p>

    <h2>API Commands</h2>

    <h2>/tree</h2>
    <dl><b>Arguments</b>
      <dd>NONE</dd>
      <dt><b>Returns</b></dt>
      <dd>full tree used in OrthoDB</dd>
    </dl>
    <dl><b>Description</b>
      <dd>This retrieves the full tree.</dd>
    </dl>
    <p><a href="tree">Example</a></p>
      
    <h2>/search</h2>
    <dl><b>Arguments</b>
      <dt>query</dt>
      <dd>full query string</dd>
      <dt>ncbi</dt>
      <dd>flag - if 0, then generic search, if 1 the query is assumed to be a NCBI gene id</dd>
      <dt>level</dt>
      <dd>NCBI taxon id of the clade</dd>
      <dt>skip</dt>
      <dd>number of hits to skip</dd>
      <dt>limit</dt>
      <dd>maximum nr of hits (cluster ids) to return</dd>
      <dt>universal</dt>
      <dd>phyloprofile filter, present in 1.0, 0.9, 0.8 of all species in the clade</dd>
      <dt>singlecopy</dt>
      <dd>phyloprofile filter, singlecopy in 1.0, 0.9, 0.8 of all species in the clade</dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>a list of clusters</dd>
    </dl>
    <dl><b>Description</b>
      <dd>This finds all cluster id's matching a given query.</dd>
    </dl>
    <p><a href="search?query=p450&limit=2&level=33208&singlecopy=0.8">Example</a></p>
    
    <h2>/blast</h2>
    <dl><b>Arguments</b>
      <dt><i>all arguments for <b>/search</b> except 'query' and 'ncbi'</i> ---</dt>
      <dt>seq</dt>
      <dd>sequence string, <b><i>without fasta-header</i></b></dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>list of OrthoDB clusters</dd>
    </dl>
    <dl><b>Description</b>
      <dd>TThis finds all cluster id's with genes matching the given sequence. The list is sorted with the best matching cluster first.</dd>
    </dl>
    <p><a href="blast?level=33208&seq=MGDSHEDTSATVPEAVAEEVSLFSTTDIVLF">Example</a></p>
    
    <h2>/group</h2>
    <dl><b>Arguments</b>
      <dt>id</dt>
      <dd>OrthoDB cluster id </dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>annotation details on the given cluster id</dd>
    </dl>
    <dl><b>Description</b>
      <dd>Retrieve detailed annotation information on the given cluster.</dd>
    </dl>
    <p><a href="group?id=EOG091G02VT">Example</a></p>
    
    <h2>/orthologs</h2>
    <dl><b>Arguments</b>
      <dt>id</dt>
      <dd>OrthoDB cluster id</dd>
      <dt>species</dt>
      <dd>optional comma-separated list of species taxid's</dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>a dictionary of tax id's, each contain a list of OrthoDB gene id's</dd>
    </dl>
    <dl><b>Description</b>
      <dd>Retrieve all genes in a given cluster, possibly filtered wrt species.</dd>
    </dl>
    <p><a href="orthologs?id=EOG091G02VT">Example</a></p>
    
    <h2>/ogdetails</h2>
    <dl><b>Arguments</b>
      <dt>id</dt>
      <dd>OrthoDB gene id</dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>detailed information on the given gene id</dd>
    </dl>
    <dl><b>Description</b>
      <dd>Retrieve further details on a given gene id.</dd>
    </dl>
    <p><a href="ogdetails?id=6087:00306f">Example</a></p>
    
    <h2>/siblings</h2>
    <dl><b>Arguments</b>
      <dt>id</dt>
      <dd>OrthodDB cluster id</dd>
      <dt>limit</dt>
      <dd>max nr of returned siblings</dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>a list of OrthoDB cluster id's</dd>
    </dl>
    <dl><b>Description</b>
      <dd>Retrieve all siblings to the given cluster.</dd>
    </dl>
    <p><a href="siblings?id=EOG091G02VT">Example</a></p>
    
    <h2>/fasta</h2>
    <dl><b>Arguments 1</b>
      <dt>id</dt>
      <dd>OrthoDB cluster id</dd>
      <dt>species</dt>
      <dd>list of NCBI species taxonomy id's</dd>
    </dl>
    <dl><b>Arguments 2</b>
      <dt><i>all arguments for <b>/search</b></i></dt>
      <dt>species</dt>
      <dd>list of NCBI species taxonomy id's</dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>sequences in fasta format</dd>
    </dl>
    <p>Note that this query is limited by a maximum of 5000 clusters. If the limit is exceeded, a page is given with basic instructions on how to retrieve the information.
    <p><a href="fasta?id=EOG091G02VT">Example</a></p>
    
    <h2>/tab</h2>
    <dl><b>Arguments</b>
      <dt><i>same arguments as for <b>/fasta</b></i></dt>
      <dt>long</dt>
      <dd>0 (default) -> without sequence ; 1 -> include sequence</dd>
    </dl>
    <dl>
      <dt><b>Returns</b></dt>
      <dd>tab-separated table of gene annotations</dd>
    </dl>
    <p><a href="tab?id=EOG091G02VT">Example</a></p>

    <!--  Google Analytics -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
      
      ga('create', 'UA-16460000-3', 'auto');
      ga('send', 'pageview', 'help_api' );
    </script>
  </body>
</html>
