<!--Robâ€™s edited version of the HELP page, implementing comments/corrections from Mathieu and Mose.Some bullet points/formatting changed when copy-pasting from the webpage, but the focus here is on correcting and updating the text not the formatting//--><html>  <head>    <style>      body {background-color:white}      h1   {color:#7D3364}      h2   {color:black}      h3   {color:#990066}    </style>    <meta http-equiv="content-type" content="text/html; charset=utf-8"><!--    <meta http-equiv="content-type" content="text/html; charset=windows-1252">//-->  </head>  <body>    <h1><b>Search Parameters</b></h1>    <h2><b>Text Search</b></h2>    <h3><b>Enter a gene name, identifier, annotation keyword, phenotype, etc.</b></h3>    <ul class="disc">      <li>OrthoDB can be queried using relevant <b>identifiers</b> of proteins, genes, OrthoDB orthologous        groups (EOG...), InterPro domains (IPR...), or Gene Ontology terms (GO:...), as well as with <b>keywords</b> associated with protein annotations.        <ul class="disc">          <li><b>Identifiers</b>: UniProtKB, Ensembl, EntrezGene, KEGG, UniGene, GenBank, RefSeq, InterPro, Gene Ontology, AphidBase, BeetleBase, FlyBase, Hymenoptera Genome Database, LepBase, SilkDB, VectorBase, wFleaBase, Mouse Genome Informatics, Saccharomyces Genome Database, etc. e.g.            '<a target="_blank" href="http://orthodb.org/?query=P38903&level=4751&species=4751">P38903</a>',            '<a target="_blank" href="http://orthodb.org/?query=CG10753&level=33208&species=33208">CG10753</a>',            '<a target="_blank" href="http://orthodb.org/?query=IPR001163&level=33208&species=33208">IPR001163</a>'          </li>          <li><b>Keyword annotations</b> in UniProtKB and Ensembl: Protein names, gene names, etc. e.g.            <a target="_blank" href="http://orthodb.org/?query=%22Probable%20small%20nuclear%20ribonucleoprotein%20Sm%20D1%22&level=33208&species=33208">"Probable small nuclear ribonucleoprotein Sm D1"</a>          </li>          <li><b>Keyword phenotypes</b>: For <i>Homo sapiens</i>, <i>Mus musculus</i>, <i>Danio rerio</i>, <i>Caenorhabditis elegans</i>, <i>Drosophila melanogaster</i>, <i>Saccharomyces cerevisiae</i> and <i>Escherichia coli</i>, you can search the database using phenotype keywords.            <ul class="small">              <li><i>Homo sapiens</i>: Human disease data from <a href="http://www.omim.org/" target="_blank">OMIM</a> (Online Mendelian Inheritance in Man) e.g. <a target="_blank" href="http://orthodb.org/?query=%22Diabetes%20mellitus%22&level=33208&species=33208">"Diabetes mellitus"</a>               </li>              <li><i>Mus musculus</i>: High-level Mammalian Phenotype Terms from <a href="http://www.informatics.jax.org/" target="_blank">MGI</a> (Mouse Genome Informatics) e.g. phenotype: <a href="http://orthodb.org/?query=%22immune%20system%22&level=33208&species=33208">"immune system"</a>              </li>              <li><i>Danio rerio</i>: Zebrafish phenotype data from <a href="http://zfin.org/" target="_blank">ZFIN</a> (The Zebrafish Model Organism Database) e.g. <a target="_blank" href="http://orthodb.org/?query=dead&level=33208&species=33208">"dead"</a></li>              <li><i>Caenorhabditis elegans</i>: Phenotypic data from <a href="http://www.wormbase.org/" target="_blank">WormBase</a> e.g. <a target="_blank" href="http://orthodb.org/?query=%22life%20span%20variant%22&level=6231&species=6231">"life span variant"</a></li>              <li><i>Drosophila melanogaster</i>: Data from <a href="http://www.flybase.org/" target="_blank">FlyBase</a> for phenotypic classes containing keywords "lethal", "sterile", and "defective", e.g. <a target="_blank" href="http://orthodb.org/?query=%22neurophysiology%20defective%22&level=32281&species=32281">"neurophysiology defective"</a></li>              <li><i>Saccharomyces cerevisiae</i>: Phenotypic data from <a href="http://www.yeastgenome.org/" target="_blank">SGD</a> (Saccharomyces Genome Database) e.g. '<a target="_blank" href="http://orthodb.org/?query=inviable&level=4893&species=4893">inviable</a>'</li>              <li><i>Escherichia coli</i>: Phenotypic data from <a href="http://www.ecogene.org/3.0/" target="_blank">EcoGene</a> and essentiality data from <a href="http://tubic.tju.edu.cn/deg/" target="_blank">DEG</a> (Database of Essential Genes)</li>            </ul>          </li>        </ul>      </li>    </ul>    <ul class="small">      <li>Logical operator <b>NOT</b> use <b>'-'</b> or <b>'!'</b>, e.g. '<a target="_blank" href="http://orthodb.org/?query=kinase%20-serine&level=33208&species=33208">kinase -serine</a>' or '<a target="_blank" href="http://orthodb.org/?query=kinase%20!tyrosine&level=33208&species=33208">kinase !tyrosine</a>'      </li>      <li>Logical operator <b>OR</b> use <b>'|'</b>, e.g. '<a target="_blank" href="http://orthodb.org/?query=protease%20|%20peptidase&level=33208&species=33208">protease | peptidase</a>'.      </li>      <li>Logical operator <b>AND</b> is implicit, i.e. '<a target="_blank" href="http://orthodb.org/?query=sodium%20transporter&level=33208&species=33208">sodium transporter</a>' actually means 'sodium AND transporter' (not quoted phrases).      </li>      <li>Use <b>quotes</b> to match a phrase literally, e.g. <a target="_blank" href="http://orthodb.org/?query=%22Cytochrome%20P450%22&level=33208&species=33208">"Cytochrome P450"</a>.      </li>      <li>Take advantage of the <b>autocomplete </b>lookup feature that offers keyword or identifier suggestions for your search.      </li>      <li>Click the <b>'Submit'</b> button (or return key) to execute the query.      </li>      <li>The OrthoDB Search Engine is powered by <b><a href="http://www.sphinxsearch.com/" target="_blank">Sphinx</a></b>.      </li>    </ul>    <h2><b>Sequence Search</b></h2>    <h3><b>Retrieve orthologous groups by homology to your query protein sequence.</b></h3>    <ul class="small">      <li>Paste your protein sequence in any format (e.g. FASTA), only one sequence will be considered at a time, and click the <b>'Submit'</b> button to execute the query.      </li>      <li>Orthologous Groups with members that have significant matches to your query sequence will be returned.      </li>      <li>The OrthoDB sequence search is powered by a modified version of <a href="http://omics.informatics.indiana.edu/mg/RAPSearch2/">RAPSearch</a>.      </li>    </ul>    <h2><b>Phyloprofile Filters</b></h2>    <h3><b>Filtering by phylogenetic presence and/or gene copy-number.</b></h3>    <ul class="small"><li>You can filter the list of Orthologous Groups returned to require them to have members in all species of the selected taxonomic node, or a fraction of them, e.g. present in all, in >90% or >80% of the species.    </li>      <li>You can filter the list of Orthologous Groups returned to require them to have only single-copy orthologs in all species of the selected taxonomic node, or a fraction of them, e.g. single-copy in all, in >90% or >80% of the species.</li>      <li>You can combine any presence filter with any copy-number filter to refine your results, e.g. present in >90% AND single-copy in >80% of species.</li>    </ul>        <h2><b>Species Selection</b></h2>    <h3><b>Tailor your search by using the expandable species tree to select a radiation point or particular sets of species.</b></h3>    <ul class="small">      <li>The expandable species tree is derived from the official <a href="http://www.ncbi.nlm.nih.gov/taxonomy">NCBI Taxonomy</a>.      </li>      <li>OrthoDB Orthologous Groups are delineated at each major radiation node of the species phylogeny, with separate root nodes for <b>Bacteria</b>, <b>Metazoa</b>, <b>Fungi</b>, and <b>Plants</b>.      </li>      <li>This hierarchy of Orthologous Groups means that considering many distantly-related species delineates fewer, more <b>general</b> (inclusive) orthologous groups containing all the descendants of the ancestral gene, while examining only sets of more closely-related species produces many <b>fine-grained</b> orthologous groups of mostly one-to-one relations.      </li>      <li><b>Expand or collapse</b> any node on the tree by clicking on the filled arrows or node names.      </li>      <li><b>Select all species</b> at a node by clicking on the unfilled box next to the node name, or <b>select specific species</b> by clicking on the unfilled box next to the species name.      </li>      <li>The appropriate <b>node to be searched</b> (most recent common ancestor of all selected species) and the <b>list of selected species to display</b> is automatically updated based on your species and/or node selections.      </li>      <li>You may also <b>add species to the list of selected species to display</b> by typing the species name in the <b>search box</b> and selecting from the autocompleted options.      </li>      <li>As you add or remove species from the expandable species tree, the <b>'Species to display'</b> box above it will automatically update to reflect your selections.      </li>    </ul>        <h2><b>Species to display</b></h2>    <h3><b>Filter your results to display only the species of interest.</b></h3>    <ul class="small">      <li>You may choose to view <b>reference species only</b> at a particular node by clicking the option in the <b>'Species to display'</b> box, and then <b>'Submit'</b>. These are selected to include species that are considered model species or are generally better studied.      </li>      <li>If you have chosen to view only a subset of species, you may easily toggle back and forth with the full species view while browsing the <b>'Orthologs by organism'</b> table using the <b>'Selected species only'</b> checkbox.      </li>      <li>You may reset your species selection using the <b>'Clear all'</b> option from the <b>'Species to display'</b> box, and then <b>'Submit'</b>.      </li>    </ul>    <p></p>    <h1><b>Results</b></h1>    <h2><b>Functional descriptions</b></h2>    <h3><b>Summary information of gene functional annotations.</b></h3>    <ul class="small">      <li><b>Gene Ontology</b> and <b>InterPro</b> attributes associated with individual member proteins provide a <b>general description</b> for the Orthologous Group as a whole.      </li>      <li>These are summed over the member proteins to indicate the most frequently occurring attributes associated with the Orthologous Group.      </li>      <li><b>Gene Ontology</b> and <b>InterPro</b> identifiers are linked to their respective sources to access further detailed information describing the Terms and Domains.      </li>      <li><b>Functional Categories</b> are provided based on mapping to <a href="http://www.ncbi.nlm.nih.gov/COG/">COG</a>-defined broad gene functional categories.      </li>    </ul>    <h2><b>Evolutionary descriptions</b></h2>    <h3><b>Summary information of the orthologous groupâ€™s evolutionary profile.</b></h3>    <ul class="small">      <li><b>Phyletic Profile</b>: a summary of the ortholog presence (from universal to species-specific) and copy-numbers (single/multi-copy counts).      </li>      <li><b>Evolutionary Rate</b>: Orthologous groups that exhibit appreciably higher or lower levels of sequence divergence are highlighted through quantification of the relative divergence among their member genes. These are computed for each orthologous group as the average of inter-species identities normalized to the average identity of all inter-species best reciprocal hits, computed from pairwise Smith-Waterman alignments of protein sequences. The relative rate is indicated by the position of the black star along the scale of slow-blue to fast-red rates.      </li>      <li><b>Gene Architecture</b>: The table shows mean, median, and standard deviation values of protein lengths and exon counts for each orthologous group, effectively describing a 'consensus' gene architecture (for those genes with available data).      </li>    </ul>    <h2><b>Orthologs by organism</b></h2>    <h3><b>Orthologs listed per species with associated annotations.</b></h3>    <ul class="small">      <li>The table view presents <b>available information for each gene</b> including gene/protein ID, UniProt ID, short description, number of amino acids (AAs), number of exons, and associated InterPro domains.      </li>      <li>If <b>additional information</b> is available for a given gene a double-arrow icon can be clicked to view an expanded association annotation list with links to source databases.      </li>      <li>Available (i.e. those mapped from UniProt) InterPro attributes are displayed for each protein member <b>ordered from the N to C terminus</b>.      </li>      <li>To reduce redundancy, where a protein exhibits two or more domains related as child=>parent, only the child attribute is displayed, while all are searchable.      </li>      <li>Click on the grey magnifying glass icon to <b>query OrthoDB for groups containing proteins with the same domains</b>.      </li>      <li>To search for <b>specific domain architectures</b>, enter an ordered list of InterPro identifiers separated with only commas into the 'Text Search' box, e.g. '<a target="_blank" href="http://orthodb.org/?query=IPR000253%2C%20IPR000719&level=33208&species=33208">IPR000253,IPR000719</a>' Forkhead-domain, Kinase-domain</li>      <li>Gene <b>synonyms</b> and recorded <b>phenotypes</b> are integrated in OrthoDB for <b>Human</b> (from <a href="http://www.omim.org/" target="_blank">OMIM</a>: Online Mendelian Inheritance in Man) and selected model organisms (<i>C. elegans</i>: <a href="http://www.wormbase.org/" target="_blank">WormBase</a>, <i>D. melanogaster</i>: <a href="http://www.flybase.org/" target="_blank">FlyBase</a>, <i>D. rerio</i>: <a href="http://zfin.org/" target="_blank">ZFIN</a> The Zebrafish Model Organism Database, <i>E. coli</i>: <a href="http://www.ecogene.org/3.0/" target="_blank">EcoGene</a>, <i>M. musculus</i>: <a href="http://www.informatics.jax.org/" target="_blank">MGI</a> Mouse Genome Informatics, and <i>S. cerevisiae</i>: <a href="http://www.yeastgenome.org/" target="_blank">SGD</a> Saccharomyces Genome Database). </li>      <li>Genes with <b>synonyms</b> <img src="static/images/synonym-icon.png" alt="S" height="14"> or <b>phenotypes</b> <img src="static/images/phenotype-icon.png" alt="S" height="14"> are linked to their respective source databases.</li>      <li>For the length (AAs) and exon counts (Exons) listed for each gene, the exclamation mark (!) indicates <b>differences from consensus</b> (left: shorter, right: longer, !: 1 stdev, !!: 2 stdev).      </li>      <li>As some species are included for complete orthology clustering while others are later mapped to defined clusters, those <b>species included only through mapping</b> are indicated with a grey M icon.      </li>    </ul>    <h2><b>Sibling groups</b></h2>    <h3><b>Related orthologous groups at the same species radiation.</b></h3>    <ul class="small">      <li>Relations among Orthologous Groups are defined according to their <b>common InterPro domain annotations</b>.      </li>      <li>The top 5 groups are listed with their percentage overlap in terms of common InterPro domains, and the complete list of related groups may be retrieved by clicking the '<b>Show all siblings</b>' link.      </li>    </ul>    <p></p>    <h1><b>Export Results</b></h1>    <h2><b>Get All Fasta / View Fasta</b></h2>    <h3><b>Displays text view of search result protein sequences in Fasta format.</b></h3>    <ul class="small">      <li><b>Get All Fasta</b> = all proteins from your selected species from all orthologous groups that match your search.</li>      <li><b>View Fasta</b> = all proteins from your selected species from the orthologous group you are viewing.</li>      <li>Group ID, gene, organism, and other useful details are contained in the header of each sequence. This information can be saved as a file by right-clicking on the link followed by "save link as...".</li>    </ul>    <h2><b>Get All as Tab Delimited / View Tab Delimited</b></h2>    <h3><b>Displays tab delimited text view of results.</b></h3>    <ul class="small">      <li><b>Get All as Tab Delimited</b> = all proteins from your selected species from all orthologous groups that match your search.</li>      <li><b>View Tab Delimited</b> = all proteins from your selected species from the orthologous group you are viewing.</li>      <li>Group ID, taxonomic level, and protein IDs are listed for each sequence. This information can be saved as a file by right-clicking on the link followed by "save link as...".</li>    </ul>    <p></p>    <h1><b>Bookmarks & Navigation</b></h1>    <h2><b>Bookmarking</b></h2>    <p>Search results can be saved by simply bookmarking the result page or saving the URL text.</p>    <h1><b>Data & Citations</b></h1>    <h2><b>Data Sources</b></h2>    <ul class="small">      <li><b>Fungi Proteomes: </b>retrieved from <a href="http://www.uniprot.org/help/uniprotkb">Uniprot</a>.</li>      <li><b>Arthropod Proteomes: </b>retrieved from        <a href="http://www.aphidbase.com/">AphidBase</a>,        <a href="http://beetlebase.org/">BeetleBase</a>,        <a href="http://flybase.org/">FlyBase</a>,        <a href="http://hymenopteragenome.org/">Hymenoptera Genome Database</a>,        <a href="http://silkworm.genomics.org.cn/silkdb/">SilkDB</a>,        <a href="http://www.vectorbase.org/index.php">VectorBase</a>,        <a href="http://wfleabase.org/">wFleaBase</a>,        <a href="http://www.ncbi.nlm.nih.gov/">NCBI</a>        and various consortia.</li>      <li><b>Vertebrate Proteomes: </b>retrieved from        <a href="http://www.ensembl.org/index.html">Ensembl</a>        release 71 and        <a href="http://www.ncbi.nlm.nih.gov/">NCBI</a>.      </li>      <li><b>Basal Metazoan Proteomes: </b>retrieved from        <a href="http://www.ensembl.org/index.html">Ensembl</a>        release 71,        <a href="http://metazoa.ensembl.org/index.html">Ensembl Metazoa</a>        release 18,        <a href="http://www.ncbi.nlm.nih.gov/">NCBI</a>        and the Joint Genome Institute        <a href="http://www.jgi.doe.gov/">JGI</a>.</li>      <li><b>Bacteria Proteomes: </b>retrieved from <a href="http://www.ncbi.nlm.nih.gov/">NCBI</a>.      </li>    </ul>    <h2><b>Data Downloads</b></h2>    <p>Download all protein sequences and orthologous group annotations for major clades <a href="?page=downloads" onclick="app.href(event, this)">here.</a></p>    <h2><b>Application programming interface (API)</b></h2>    <p>For information on how to access OrthoDB results programmatically, please visit our <a href="?page=api" onclick="app.href(event, this)">API help</a>    </p>        <h2><b>Citing OrthoDB</b></h2>    <h3><b>If you have used OrthoDB in your work, please use the corresponding citation below.</b></h3>    <ul class="small">      <li><b>OrthoDB v8: update of the hierarchical catalog of orthologs and the underlying free software.</b></li>      <p>Kriventseva <i>et al</i>, Nucleic Acids Research, January 2015, PMID:        <a href="http://www.ncbi.nlm.nih.gov/pubmed/25428351">25428351</a>      </p>      <li><b>OrthoDB: a hierarchical catalog of animal, fungal and bacterial orthologs.</b></li>      <p>Waterhouse <i>et al</i>, Nucleic Acids Research, January 2013, PMID:        <a href="http://www.ncbi.nlm.nih.gov/pubmed/23180791">23180791</a>      </p>      <li><b>OrthoDB: the hierarchical catalog of eukaryotic orthologs in 2011.</b></li>      <p>Waterhouse <i>et al</i>, Nucleic Acids Research, January 2011, PMID:        <a href="http://www.ncbi.nlm.nih.gov/pubmed/20972218">20972218</a>      </p>      <li><b>OrthoDB: the hierarchical catalog of eukaryotic orthologs.</b></li>      <p>Kriventseva <i>et al</i>, Nucleic Acids Research, January 2008, PMID:        <a href="http://www.ncbi.nlm.nih.gov/pubmed/17947323">17947323</a>      </p>    </ul>    <h2><b>OrthoDB Citations</b></h2>    <h3><b>Click the titles below for citation lists of each OrthoDB publication</b></h3>    <ul class="small">      <li><a href="http://scholar.google.ch/scholar?cites=6522594302406737801&as_sdt=2005&sciodt=0,5&hl=en">2015: OrthoDB v8: update of the hierarchical catalog of orthologs and the underlying free software.</a></li>      <li><a href="http://scholar.google.ch/scholar?cites=1980651359411099267&as_sdt=2005&sciodt=0,5&hl=en">2013: OrthoDB: a hierarchical catalog of animal,fungal and bacterial orthologs.</a></li>      <li><a href="http://scholar.google.ch/scholar?cites=7988060515234758019&as_sdt=2005&sciodt=0,5&hl=en">2011: OrthoDB: the hierarchical catalog of eukaryotic orthologs in 2011.</a></li>      <li><a href="http://scholar.google.ch/scholar?cites=16780328197132691044&as_sdt=2005&sciodt=0,5&hl=en">2008: OrthoDB: the hierarchical catalog of eukaryotic orthologs.</a></li>    </ul>    <h2><b>Previous OrthoDB Releases</b></h2>    <ul class="small">      <li><a href="http://cegg.unige.ch/orthodb">OrthoDB8 2014</a>: 61 vertebrates, 87 arthropods, 227 fungi, 12 basal metazoans, and 2627 bacteria</li>      <li><a href="http://cegg.unige.ch/orthodb7">OrthoDB7 2013</a>: 64 vertebrates, 57 arthropods, 175 fungi, 14 basal metazoans, and 1115 bacteria</li>      <li><a href="http://cegg.unige.ch/orthodb6">OrthoDB6 2012</a>: 52 vertebrates, 45 arthropods, 142 fungi, 13 basal metazoans, and 1115 bacteria</li>      <li><a href="http://cegg.unige.ch/orthodb5">OrthoDB5 2011</a>: 48 vertebrates, 33 arthropods, 73 fungi, and 12 basal metazoans</li>      <li><a href="http://cegg.unige.ch/orthodb4">OrthoDB4 2010</a>: 44 vertebrates, 25 arthropods, 46 fungi, and 5 basal metazoans</li>      <li><a href="http://cegg.unige.ch/orthodb3">OrthoDB3 2009</a>: 40 vertebrates, 23 arthropods, and 32 fungi</li>      <li><a href="http://cegg.unige.ch/orthodb2">OrthoDB2 2008</a>: 15 vertebrates, 20 arthropods, and 23 fungi</li>    </ul>        <h2><b>Contact</b></h2>    <p>Please email support[at]orthodb.org</p><p></p></body></html>